privateca/certificate_authority_subordinate/main.tf

```diff
@@ -51,12 +50,14 @@ resource "google_privateca_certificate_authority" "root_ca" {
   key_spec {
     algorithm = "RSA_PKCS1_4096_SHA256"
   }
+  // Root CA should be valid indefinetly e.g. 99 years
+  lifetime = "${99 * 365 * 24 * 3600}s"
 }
 
 resource "google_privateca_certificate_authority" "default" {
   // This example assumes this pool already exists.
   // Pools cannot be deleted in normal test circumstances, so we depend on static pools
-  pool                     = "my-pool"
+  pool                     = "my-sub-pool"
   certificate_authority_id = "my-certificate-authority-sub"
   location                 = "us-central1"
   deletion_protection      = false # set to true to prevent destruction of the resource
```

Comment:
_(sorry for the misplaced comment -- I can't figure out how to comment on the actual lines I want to mention, which weren't modified here)_

Could you also make the following changes for the sub CA:
- Remove the `subject_alt_name`
- Get rid of all `base_key_usage` values except for `cert_sign` and `crl_sign`.
- Get rid of the `extended_key_usage` block entirely
- Update the lifetime from 1 day (86400s) to 5 years (`"{5 * 365 * 24 * 3600}"`)
- Update the signing key algorithm to use `RSA_PKCS1_2048_SHA256` (more efficient for use in a sub CA)

Reply:
done, also took the freedom to replace HashiCorp with ACME
eku block is empty, someone has to check what the provider actually makes out of it

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/privateca/CertificateAuthority.yaml#L404 there it's requiring the EKU block